### PR TITLE
content/blogs: update iot.md with style edits

### PR DESCRIPTION
**Pull request by odeke-em**
* Affix the various `shell` code identifier to shell code
* Affix the JSON code identifier to JSON code
as well as appropriately quote and delimit its fields
* Remove the class "resized-logo" to ensure the images
are full-sized
* A space between `Filter` and method
* "Google, Inc" --> "Google, LLC"

### CODE REVIEW

File: content/blogs/_index.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 title: "Blog"
 date: 2018-07-30T18:34:00-07:00
 draft: false
-class: "resized-logo"
+weight: 90
```
Comment (odeke-em) [inline content/blogs/_index.md]: ### Before
screen shot 2018-08-09 at 4 04 41 pm


### After
screen shot 2018-08-09 at 4 05 24 pm

File: content/blogs/iot.md [hunk 2]
```diff
@@ -71,7 +70,7 @@ In the first system, Raspberry Pi collects data from multiple sensors and then c
 
 ###### Step 1:
 
-```
+```shell
```
Comment (odeke-em) [inline content/blogs/iot.md]: ### Before
screen shot 2018-08-09 at 4 02 23 pm


### After
screen shot 2018-08-09 at 4 02 45 pm

File: content/blogs/iot.md [hunk 3]
```diff
@@ -80,7 +79,7 @@ During this step, you would install the required dependencies and configure the
 
 ###### Step 2:
 
-```
+```shell
```
Comment (odeke-em) [inline content/blogs/iot.md]: ### Before
screen shot 2018-08-09 at 4 03 00 pm

### After
screen shot 2018-08-09 at 4 03 29 pm

File: content/blogs/iot.md [hunk 4]
```diff
@@ -132,12 +131,12 @@ To decouple the master and slave nodes in the System II, we design a robust and
 
 The protocol defines the basic format of messages exchanged between the master and slave nodes. One example is as below:
 
-```
+```json
```
Comment (odeke-em) [inline content/blogs/iot.md]: ### Before
screen shot 2018-08-09 at 4 03 58 pm



### After
screen shot 2018-08-09 at 4 04 16 pm

File: content/blogs/iot.md [hunk 5]
```diff
@@ -153,7 +152,7 @@ Moreover,  the protocol defines the message exchange pattern between master and
 
 Configuration for the system II is very similar to the one in the System I. The only difference is the first step.
 
-```
+```shell
```
Comment (odeke-em) [inline content/blogs/iot.md]: ditto

File: content/blogs/iot.md [hunk 6]
```diff
@@ -35,7 +34,7 @@ Our application is a prototype which demonstrates how we can utilize OpenCensus
 
 #### Installation
 
-```
+```shell
```
Comment (odeke-em) [inline content/blogs/iot.md]: ### Before
screen shot 2018-08-09 at 3 56 46 pm

### After
screen shot 2018-08-09 at 4 01 47 pm

**Comment by odeke-em** [comment]
Awesome, thanks for the review @songy23! I'll merge this as the day is coming to an end and folks are going away from their code editors.

**Comment by hbgxsm95** [comment]
Looks great! Thanks!

**Pull request merged**